lm/lm_image.py

```diff
@@ -0,0 +1,120 @@
+__all__ = ["LmImageJob"]
+
+import shutil
+
+from sisyphus import *
+from i6_core import rasr
+from i6_core import util
+
+Path = setup_path(__package__)
+
+
+class LmImageJob(rasr.RasrCommand, Job):
+    """
+    pre-compute LM image without generating global cache
+    """
+
+    def __init__(
+        self,
+        crp,
+        extra_config=None,
+        extra_post_config=None,
+        encoding="utf-8",
+        renormalize=False,
```

Comment:
What does `renormalize` do here? I mean it will somehow normalize the LM, but why do we have it as parameter in this Job, whos job it is to compute the image to an existing LM?

Will the LM file be modified? Will only the image be normalized? Is the image now inconsistent to the text file?

In the favor of "one Job should do one thing" I would argue that we can have a NormalizeLmJob if need be, and that the LmImageJob only does write out the image.

Reply:
Sounds fair, I will remove this parameter.